contrib/pg_tde/documentation/docs/how-to/enforcement.md

```diff
@@ -0,0 +1,14 @@
+# Encryption Enforcement
+
+For `pg_tde`, encryption enforcement ensures that only encrypted storage is allowed for specific operations, tables, or the entire database. It prevents the accidental creation of unencrypted tables or indexes in environments where encryption is required for compliance, security, or policy enforcement.
+
+## Enforcing encryption usage
+
+Use the following techniques to enforce the secure use of `pg_tde`.
+
+### 1. Use pg_tde.enforce_encryption
+
+The [`pg_tde.enforce_encryption`](../variables.md/#pg_tdeenforce_encryption) option prevents non encrypted tables to be created.
+
+!!! note
+    This option prevents unauthorized users from creating unencrypted tables or indexes, ensuring encryption compliance. Superusers retain the privilege to create unencrypted tables as necessary.
```

Comment:
No, superusers can override the variable at a session level, they also can't create unencrypted tables if the variable is enabled.